src/CommunityToolkit/Xamarin.CommunityToolkit.UnitTests/Behaviors/AnimationBehavior_Tests.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Threading.Tasks;
+using NUnit.Framework;
+using Xamarin.CommunityToolkit.Behaviors;
+using Xamarin.CommunityToolkit.UnitTests.Mocks;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.UnitTests.Behaviors
+{
+	public class AnimationBehavior_Tests
+	{
+		[SetUp]
+		public void SetUp() => Device.PlatformServices = new MockPlatformServices();
+
+		[Test]
+		public void AnimateCommandStartsAnimation()
+		{
+			var mockAnimation = new MockAnimationType();
+
+			var behavior = new AnimationBehavior
+			{
+				AnimationType = mockAnimation
+			};
+
+			new Label
+			{
+				Behaviors =
+ 				{
+ 					behavior
+ 				}
+			};
+
+			behavior.AnimateCommand.Execute(null);
+
+			Assert.That(mockAnimation.HasAnimated, Is.True);
```

Comment:
Assert.IsTrue(mockAnimation.HasAnimated)

?

Reply:
Do you want me to resolve these now that I have actioned them?